index.html

```diff
@@ -97,7 +97,122 @@ <h2>Software Developer</h2>
 
             </ul>
         </section>
+        <section class="detailContent">
+            <h2 class="worktitile"> MY RECENT WORK</h2>
+            <ul class="works">
+               
+                <li class="workCard">
+                    <img src="Images/projectOne.png" alt="projectone">
+                    <h3>Multi-post Stories</h3>
+                    <p>A daily selection of privately personalized reads; no accounts or
+                        sign-ups required. has been the industry's standard dummy text ever
+                        since the 1500s, when an unknown printer took a standard dummy text.
+                    </p>
+                    <ul class="workTechs">
+                        <li>css</li> 
+                        <li>html</li>
+                        <li>Bootstrap</li>
+                        <li>Ruby</li>
```

Comment:
### Your project :bust_in_silhouette:
![missing side border](https://user-images.githubusercontent.com/93039359/176909693-15dd7893-424e-4785-b289-056ceb79c5de.png)

### Figma Template :+1:
![side borders fix](https://user-images.githubusercontent.com/93039359/176909982-3f4862f1-fabd-4a8f-90d6-ba8e17e0c59d.png)



- [ ] Please compare both images and you will find that you are missing the side borders. Please add these side borders :+1: Please change it in all the cards :+1: